packages/api/get-service-stops.ts

```diff
@@ -24,56 +24,63 @@ export const getQueryInput = (event: APIGatewayEvent): ServiceStopsQueryInput =>
     };
 };
 
-export const formatStops = (stops: ServiceStops): Stops => {
-    return stops
-        .flatMap((stop) => {
-            const stopArray: Stops = [];
+export const flattenStops = (stops: ServiceStops): ServiceStop[] => {
+    return stops.flatMap((stop) => {
+        const stopArray: ServiceStop[] = [];
+        if (stop.fromStatus === "active") {
+            stopArray.push({
+                id: stop.fromId,
+                atcoCode: stop.fromAtcoCode,
+                naptanCode: stop.fromNaptanCode,
+                commonName: stop.fromCommonName,
+                street: stop.fromStreet,
+                indicator: stop.fromIndicator,
+                bearing: stop.fromBearing,
+                nptgLocalityCode: stop.fromNptgLocalityCode,
+                localityName: stop.fromLocalityName,
+                parentLocalityName: stop.fromParentLocalityName,
+                longitude: stop.fromLongitude,
+                latitude: stop.fromLatitude,
+                stopType: stop.fromStopType,
+                busStopType: stop.fromBusStopType,
+                timingStatus: stop.fromTimingStatus,
+                administrativeAreaCode: stop.fromAdministrativeAreaCode,
+                status: stop.fromStatus,
+                direction: stop.direction || "",
+                fromSequenceNumber: stop.fromSequenceNumber || "",
```

Comment:
Maybe just call this `sequenceNumber` in the response

Reply:
done